internal/net/control/control.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/vdaas/vald/internal/log"
 )
 
+// SocketController represent the socket listener controller.
```

Comment:
I think that "represents" is more correct representation grammatically.

Reply:
```suggestion
// SocketController represents the socket listener controller.
```